service_crategen/src/commands/generate/codegen/rest_response_parser.rs

```diff
@@ -80,13 +80,13 @@ fn parse_headers_map(member_name: &str, member: &Member, required: bool) -> Stri
         format!("result.{} = Some(values);", member_name.to_snake_case())
     };
 
-    format!("let mut values = ::std::collections::HashMap::new();
+    format!(r#"let mut values = ::std::collections::HashMap::new();
```

Comment:
I think just in the interest of keeping the service generator code consistent, it would be preferable to stick to non-raw template strings.